tests/test_config_node.py

```diff
@@ -94,3 +96,20 @@ def test_add_cylc_install_to_rose_conf_node_opts(rose_conf, cli_conf, expect):
         'already in `rose-suite.conf`.'
     )]
     assert result.state == ''
+
+
+def test_dump_rose_log(monkeypatch, tmp_path):
+    # Pin down the results of the function used to provide a timestamp.
+    def fake(*arg, **kwargs):
+        return '18151210T0000Z'
+    monkeypatch.setattr(
+        DateTimeOperator, 'process_time_point_str', fake
```

Comment:
A compact way of doing this would be `lambda *a, **k: '18151210T0000Z'` instead of defining `fake`. But it doesn't matter

Reply:
Tidier.